### PR TITLE
Fix `Verify User Can Validate Scale To Zero` test case

### PR DESCRIPTION
**Pull request by hdefazio**
Add the `-connect-timeout` flag to the grpcurl command in `Query Model With GRPCURL`

Pass connect_timeout=300 for ODS-2379 in order to address [RHOAIENG-10717](https://issues.redhat.com/browse/RHOAIENG-10717)

Tested and passed [here](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/devops/job/rhoai-test-flow/326)

### CODE REVIEW

File: ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_llm_tgis.robot [hunk 1]
```diff
@@ -335,8 +335,8 @@ Verify User Can Validate Scale To Zero    # robocop: off=too-long-test-case,too-
     ...    namespace=${test_namespace}    runtime=${TGIS_RUNTIME_NAME}    exp_replicas=${2}
     Wait For Pods To Be Terminated    label_selector=serving.kserve.io/inferenceservice=${flan_model_name}
     ...    namespace=${test_namespace}
-    Query Model Multiple Times    model_name=${flan_model_name}    runtime=${TGIS_RUNTIME_NAME}    n_times=1
-    ...    namespace=${test_namespace}
+    Query Model Multiple Times    model_name=${flan_model_name}    n_times=1    connect_timeout=300
+    ...    namespace=${test_namespace}    runtime=${TGIS_RUNTIME_NAME}
```
Comment (lugi0) [inline ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_llm_tgis.robot:339]: Could you also add this to the test in `ods-ci/ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_llm.robot`?

Comment (hdefazio) [reply ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_llm_tgis.robot:339]: Yes I can - side note, is there a reason why that one directly calls `Query Model With GRPCURL`?

Comment (hdefazio) [reply ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_llm_tgis.robot:339]: Just updated!

**Review by lugi0 — APPROVED** [review]
LGTM, have you tested this with Jenkins?

**Comment by hdefazio** [comment]
@lugi0 Yes, I just updated the description to link the passing test

**Review by mwaykole — APPROVED** [review]

**Pull request merged**